docs/collections/_policies/json-format.md

```diff
@@ -931,7 +932,16 @@ JSON representation
 
 #### `like` {#JsonExpr-like}
 
-The value of this key is an object with keys `left` and `pattern`.  The left key is itself an [JsonExpr object](#JsonExpr-objects), while the `pattern` key is any string.
+The value of this key is an object with keys `left` and `pattern`.
+The left key is itself an [JsonExpr object](#JsonExpr-objects), while the `pattern` key is an array composed of pattern elements.
+A pattern element can be one of either
+  - the string `Wildcard`
+  - a [`Literal`](#JsonExpr-Literal)
```

Comment:
an object with a single key `Literal`?

Reply:
I've added that, and also defined the value to be a string here. I didn't think we needed a `Literal` object section but we could restore it.